### PR TITLE
feat: login/signup with functionality

### PR DESCRIPTION
**Pull request by vanshitalavange**
This PR includes:
- login page
- signup page
- functionality of login signup

Preview:
https://pencil-git-login-signup-vanshitalavange.vercel.app/

### CODE REVIEW

**Review by Bharatrathod-81 — APPROVED** [review]
clean code and perfect file structure

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,13 @@
...
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Header />
+      <Router />
```
Comment (ashishnnnnn) [inline src/App.js:9]: Liked the use of semantic components

File: src/pages/Auth/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                className="align-center"
+                onClick={() => {
+                  setShowPassword({ ...showPassword, confirmPassword: false });
+                }}
+              />
+            )}
+          </div>
+        </div>
+        <p className="text-centered">{helperMsg}</p>
+        <div className="btn icon-btn-container icon-btn-container-logout btn-verify justify-center btn-primary">
```
Comment (ashishnnnnn) [inline src/pages/Auth/Signup.jsx:152]: Nice use of utility classes.

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import "./Home.css";
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../contexts";
+export const Home = () => {
+  const navigate = useNavigate()
+  const { userState: { loginStatus } } = useAuth()
+  return (
+    <section className="hero flex-row justify-center flex-wrap">
+      <img className="hero-banner" src="assets/banner.png" />
```
Comment (ashishnnnnn) [inline src/pages/Home/Home.jsx]: Alt tag of img can be given here..

**Review by ashishnnnnn — APPROVED** [review]

**Pull request merged**